src/main/java/com/revature/controllers/QuestionController.java

```diff
@@ -40,4 +44,95 @@
 		return ResponseEntity.ok(questionLogic.searchQuestions(pageable,searchText, subject));
 	}
 	
+//QUESTIONPOOL CUD------------------------------------
+	@RequestMapping(method = RequestMethod.POST, value="")
+	public ResponseEntity<QuestionPool> addQuestion(@RequestBody QuestionPool question){
+		
+		return ResponseEntity.ok(questionLogic.createQuestion(question));
+		
+	}
+	
+	@RequestMapping(method = RequestMethod.PUT, value = "/update/{id}")
+	public ResponseEntity<QuestionPool> updateQuestion(@RequestBody QuestionPool question, @PathVariable ("id") Integer id){
+		
+		QuestionPool currQuestion = questionLogic.getQuestionById(id);
+		
+		if(currQuestion == null){
+			System.out.println("Question with id " + id + " not found");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal " not found" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1192)